Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -551,15 +551,13 @@ - (void)_appendOpenActionsForURL:(NSURL *)url actions:(NSMutableArray *)defaultA
 
     if (elementInfo.type == _WKActivatedElementTypeImage || elementInfo._isImage) {
 #if ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
-        if ([_delegate respondsToSelector:@selector(actionSheetAssistantShouldIncludeCopySubjectAction:)] && [_delegate actionSheetAssistantShouldIncludeCopySubjectAction:self]) {
-            // FIXME (rdar://88834304): This should be additionally gated on the relevant VisionKit SPI.
+        if ([_delegate respondsToSelector:@selector(actionSheetAssistantShouldIncludeCopySubjectAction:)] && [_delegate actionSheetAssistantShouldIncludeCopySubjectAction:self])
             [defaultActions addObject:[_WKElementAction _elementActionWithType:_WKElementActionTypeCopyCroppedImage info:elementInfo assistant:self]];
-        }
 #endif
 #if ENABLE(IMAGE_ANALYSIS)
-        if ([_delegate respondsToSelector:@selector(actionSheetAssistant:shouldIncludeShowTextActionForElement:)] && [_delegate actionSheetAssistant:self shouldIncludeShowTextActionForElement:elementInfo])
```

Comment:
does this actually need to change?

Reply:
It doesn't, but I wanted to clean this up while I was touching this code since the element is unused anyways. I can split this out into a followup if you prefer.